src/assets/toolkit/styles/utils/text.css

```diff
@@ -1,6 +1,16 @@
 @import "suitcss-utils-text";
 
 /**
+ * Make responsive SUIT CSS text-alignment utilities
+ */
+
+@media(--sm-viewport) {
+  .u-sm-textLeft {
```

Comment:
This goes for all new utilities, but make sure they're added to the documentation prior to any merge.